soc/realtek/ec/rts5912/soc.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com> / Titan Chen
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <soc.h>
+
+#if defined(CONFIG_RTS5912_ON_ENTER_CPU_IDLE_HOOK)
+bool z_arm_on_enter_cpu_idle(void)
+{
+	/* Returning false prevent device goes to sleep mode */
+	return false;
+}
+#endif
+
+void z_arm_platform_init(void)
+{
+}
```

Comment:
?

Reply:
Previously we turned on Jtag port at this function but now we have moved the Jtag feature into dts.

We will delete this part.